scripts/test/hyriseBenchmarkTPCH_test.py

```diff
@@ -103,6 +104,17 @@ def main():
   close_benchmark(benchmark)
   check_exit_status(benchmark)
 
+  visualization_file = 'TPC-H_06-PQP.svg'
+  if not os.path.isfile(visualization_file):
+    print ("ERROR: Cannot find visualization file " + visualization_file)
+    sys.exit(1)
+  with open(visualization_file) as f:
+    # Check whether the (a) the GetTable node exists and (b) the chunk count is correct for the given scale factor
+    if not '/4 chunk(s)' in f:
+      print ("ERROR: Did not find expected pruning information in the visualization file")
```

Comment:
The error does not really relate to the check.

Reply:
Why not? We are looking for the text that is part of the node description.

Reply:
Hmm .. ok